label_studio_sdk/project.py

```diff
@@ -184,13 +184,23 @@ def assign_annotators(self, users, tasks_ids):
             Dict with counter of created assignments
 
         """
-        payload = {
-            'users': [user.id for user in users],
-            'selectedItems': {'all': False, 'included': tasks_ids},
-            'type': 'AN',
-        }
-        response = self.make_request('POST', f'/api/projects/{self.id}/tasks/assignees', json=payload)
-        return response.json()
+
+        # chunk tasks in batch size
+        def chunk(tasks, batch_size):
```

Comment:
please rewrite it with 
```
def chunks(lst, n):
    """Yield successive n-sized chunks from lst."""
    for i in range(0, len(lst), n):
        yield lst[i:i + n]
```
and move outside this function (it could be placed in something like `utils`

Reply:
fixed